app/src/main/res/layout/layout_attachments.xml

```diff
@@ -28,46 +28,41 @@
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toTopOf="parent"
         app:layout_goneMarginTop="@dimen/marginStandardSmall"
         tools:listitem="@layout/item_attachment" />
 
-    <ImageView
-        android:id="@+id/attachmentsIcon"
-        android:layout_width="@dimen/mediumIconSize"
-        android:layout_height="@dimen/mediumIconSize"
-        android:layout_marginStart="@dimen/marginStandardMedium"
-        android:importantForAccessibility="no"
-        android:src="@drawable/ic_attachment"
-        app:layout_constraintBottom_toBottomOf="@id/attachmentsSizeText"
+    <LinearLayout
+        android:id="@+id/attachmentsInfo"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:paddingHorizontal="@dimen/marginStandardMedium"
+        android:clickable="true"
+        android:focusable="true"
+        android:background="?attr/selectableItemBackground"
+        android:paddingVertical="@dimen/marginStandardSmall"
+        app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toTopOf="@id/attachmentsSizeText"
-        app:tint="@color/iconColor" />
+        app:layout_constraintTop_toBottomOf="@+id/attachmentsRecyclerView">
 
-    <TextView
-        android:id="@+id/attachmentsSizeText"
-        style="@style/BodySmall.Secondary"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:layout_marginHorizontal="@dimen/marginStandardVerySmall"
-        app:layout_constraintEnd_toStartOf="@id/attachmentsDownloadAllButton"
-        app:layout_constraintStart_toEndOf="@id/attachmentsIcon"
-        app:layout_constraintTop_toBottomOf="@id/attachmentsRecyclerView"
-        tools:text="4 pièces jointes (3 MB)" />
+        <TextView
+            android:id="@+id/icon"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            app:drawableStartCompat="@drawable/ic_attachment"
+            app:tint="@color/iconColor" />
 
-    <com.google.android.material.button.MaterialButton
-        android:id="@+id/attachmentsDownloadAllButton"
-        style="@style/TextButton"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:minHeight="0dp"
-        android:text="@string/buttonDownloadAll"
-        app:layout_constraintBottom_toBottomOf="@id/attachmentsSizeText"
-        app:layout_constraintStart_toEndOf="@id/attachmentsSizeText"
-        app:layout_constraintTop_toTopOf="@id/attachmentsSizeText" />
+        <TextView
+            android:id="@+id/attachmentsSizeText"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="@dimen/marginStandardSmall"
+            style="@style/Body.Secondary"
```

Comment:
reformat this file with control+command+L 
The style attribute should always be just under the id one